Applications/Spire/Source/UiViewer/StandardUiProperties.cpp

```diff
@@ -156,7 +156,7 @@ std::shared_ptr<TypedUiProperty<Money>> Spire::make_standard_property<Money>(
   return std::make_shared<StandardUiProperty<Money>>(std::move(name), value,
     [] (QWidget* parent, StandardUiProperty<Money>& property) {
       auto setter = new QDoubleSpinBox(parent);
-      setter->setMinimum(std::numeric_limits<double>::min());
+      setter->setMinimum(std::numeric_limits<double>::lowest());
```

Comment:
Niiccce.

Reply:
I have to give credit to @shaobinspire for pointing this out.